src/utils/validIdList.js

```diff
@@ -0,0 +1,22 @@
+const validIdListRegex = /(([A-Za-z]+[\w\-:.]*)\s*)*/;
```

Comment:
it'd be nice to capitalize all the initialisms (`ID` preferred over `Id`)

Reply:
Sounds good